feat: add field annotation

WhiteMinds opened the pull request:
Ref: https://github.com/Magickbase/ckb-explorer-public-issues/issues/269

Keith-CY commented on src/components/HelpTip/index.module.scss:
```diff
@@ -0,0 +1,5 @@
+    width: 15px;
+    height: 15px;
+    margin: 0 5px;
+}
```
```suggestion
}

```

Keith-CY commented:
I didn't find the link to design draft in issue https://github.com/Magickbase/ckb-explorer-public-issues/issues/269

But I found that the color of hyperlink in tooltip matches the primary color in the screenshot at https://github.com/Magickbase/ckb-explorer-public-issues/issues/269#issuecomment-1503028034

Now the color of hyperlink in tooltip is the default one
![image](https://github.com/Magickbase/ckb-explorer-frontend/assets/7271329/f98afb28-2edb-42e4-91a0-2c164422e28a)

Please check

WhiteMinds commented:
> I didn't find the link to design draft in issue [Magickbase/ckb-explorer-public-issues#269](https://github.com/Magickbase/ckb-explorer-public-issues/issues/269)

There is no design draft for this issue. I discussed with @Sven-TBD on Discord to determine how to implement it. Now I have fixed the link color issue.

Additionally, I changed the base of the PR to #26 and ran the pre-commit script again.

Keith-CY commented:
@PainterPuppets @zhangyouxin @Daryl-L please have a review

WhiteMinds commented:
Rebased.

FrederLu commented:
When the mouse is scrolled, the prompt information on the pop-up window will be displaced. Is it better to remain unaffected.
<img width="1046" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/57f545fa-4cf2-49c6-9660-f249bae08826">
It was found that the arrow in the prompt information on the mobile terminal pointed unclearly.
<img width="318" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/4b591c5b-2130-43fb-af12-f67eeac65cda">

WhiteMinds commented:
@FrederLu Fixed, please check.
